backend/internal/adapter/external/qiita_client.go

```
@@ -0,0 +1,141 @@
+// fetchPage は 1 ページ分の記事を取得する。レスポンスの本文はこの中で閉じる。
+func (c *qiitaClient) fetchPage(ctx context.Context, username string, page int) ([]qiitaAPIArticle, error) {
+	query := url.Values{}
+	query.Set("page", fmt.Sprint(page))
+	query.Set("per_page", fmt.Sprint(qiitaPerPage))
+	endpoint := fmt.Sprintf(qiitaItemsURL, url.PathEscape(username)) + "?" + query.Encode()
+
+	resp, err := c.get(ctx, endpoint)
+	if err != nil {
+		return nil, domain.NewError(domain.ErrCodeServiceUnavailable, "Qiita記事の取得に失敗", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)